go/v1beta1/storage/elasticsearch_test.go

```diff
@@ -1406,6 +1406,259 @@ var _ = Describe("elasticsearch storage", func() {
 		})
 	})
 
+	Context("creating a batch of Grafeas notes", func() {
+		var (
+			expectedErrs       []error
+			actualErrs         []error
+			actualNotes        []*pb.Note
+			expectedNotes      map[string]*pb.Note
+			expectedNotesIndex string
+		)
+
+		// BeforeEach configures the happy path for this context
+		// Variables configured here may be overridden in nested BeforeEach blocks
+		BeforeEach(func() {
+			expectedNotesIndex = fmt.Sprintf("%s-%s-%s", indexPrefix, expectedProjectId, "notes")
+			expectedNotes = generateTestNotesMap(gofakeit.Number(2, 5))
+			for i := 0; i < len(expectedNotes); i++ {
+				expectedErrs = append(expectedErrs, nil)
+			}
+
+			transport.preparedHttpResponses = []*http.Response{
+				{
+					StatusCode: http.StatusOK,
+					Body:       createEsMsearchNoteResponse(expectedNotes, expectedErrs),
+				},
+				{
+					StatusCode: http.StatusOK,
+					Body:       createEsBulkNoteIndexResponse(expectedNotes, expectedErrs),
+				},
+			}
+		})
+
+		// JustBeforeEach actually invokes the system under test
+		JustBeforeEach(func() {
+			notes := deepCopyNotes(expectedNotes)
+
+			transport.preparedHttpResponses[0].Body = createEsMsearchNoteResponse(notes, expectedErrs)
+			transport.preparedHttpResponses[1].Body = createEsBulkNoteIndexResponse(notes, expectedErrs)
```

Comment:
isn't this setup already happening in `BeforeEach`?  we want to avoid this setup here because there is no way for child contexts to override this.